src/clubhouse.js

```diff
@@ -0,0 +1,33 @@
+const {Gio} = imports.gi;
+
+const ClubhouseIface =
+'<node> \
```

Comment:
For future reference, use a backtick string for this, then you don't have to escape the newlines.